db/migrate/20210307142336_add_phone_number_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPhoneNumberToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :phone_number, :integer
```

Comment:
Phone numbers should be stored as a string, so it would be great if you could change that

Reply:
agree